modules/brainbrowser/README.md

```diff
@@ -0,0 +1,39 @@
+# Brainbrowser
+
+## Purpose
+
+Brainbrowser is intended to allow users to visualize candidate scan session MINC 
+volumes collected for a study.
+
+## Intended Users
+
+The two primary types of users are:
+1. Imaging specialists doing online QC
+2. Project radiologists reporting incidental findings.
+
+
+## Scope
+
+Brainbrowser allows navigating volumes in sagittal, coronal, and transverse 
+slices. Multiple volumes can be displayed simultaneously and coordinates
+synchronized across the multiple volumes, if desired. Image brightness and 
+intensity can be set by the user.
+A `Play` button is also available to make feasible the navigation through the 
+time dimension of a 4-Dimensional image (such as resting state fMRI scan). 
+A measurement tool is also available allowing radiologist to report measurements
+of an incidental finding.
+
+
+## Permissions
```

Comment:
@driusan 

I know we can type the url directtly to access brainbrowser, but a user should theoretically go to it through imaging browser which is why I thought I would link to that section in Imaging Browser. Do you think it is correct to do it this way?